card/db/migrate_core_cards/data/bootstrap_layout.json

```diff
@@ -47,10 +47,35 @@
+      },
+      {
+        "name": "Default Layout",
         "type": "Layout",
-        "content": "<!DOCTYPE HTML>\r\n<html>\r\n  <head>\r\n<!-- Latest compiled and minified CSS -->\r\n<link rel=\"stylesheet\" href=\"https://maxcdn.bootstrapcdn.com/bootstrap/3.3.2/css/bootstrap.min.css\">\r\n\r\n<!-- Optional theme -->\r\n<link rel=\"stylesheet\" href=\"https://maxcdn.bootstrapcdn.com/bootstrap/3.3.2/css/bootstrap-theme.min.css\">\r\n\r\n<!-- Latest compiled and minified JavaScript -->\r\n<script src=\"https://maxcdn.bootstrapcdn.com/bootstrap/3.3.2/js/bootstrap.min.js\"></script>\r\n    {{*head}}\r\n\r\n  </head>\r\n  \r\n  <body>\r\n    <nav id=\"menu\" class=\"navbar navbar-default navbar-static-top\">\r\n     <div class=\"container\">\r\n        <ul class=\"nav navbar-nav\">\r\n         <li>[[/ | Home]]</li>\r\n         <li>[[/recent | Recent]]</li>\r\n         <li>{{*navbox}}</li>\r\n       </ul>   \r\n       <ul class=\"nav navbar-nav navbar-right\"> \r\n          <li>{{*account links}}</li>\r\n       </ul>\r\n      </div>\r\n    </nav>\r\n    \r\n    <div id=\"primary\">\r\n      {{_main}}\r\n    </div>\r\n    \r\n    <div id=\"secondary\">\r\n      {{*sidebar}}\r\n    </div>\r\n  </body>\r\n</html>"
+        "content": "<!DOCTYPE HTML>\r\n<html>\r\n  <head>\r\n    {{*head}}\r\n  </head>\r\n  \r\n  <body>\r\n    <div id=\"menu\">\r\n      {{*page header}}\r\n    </div>\r\n    \r\n    <div id=\"primary\">\r\n      {{_main}}\r\n    </div>\r\n    \r\n    <div id=\"secondary\">\r\n      {{*sidebar}}\r\n    </div>\r\n  </body>\r\n</html>"
+      },
+      {
+        "name": "*all+*script",
+        "type": "Pointer",
+        "content": "[[script: jquery]]\n[[script: tinymce]]\n[[script: slot]]\n[[script: card menu]]\n[[script: jquery helper]]\n[[script: bootstrap]]\n[[script: ace]]"
+      },
+      {
+        "name": "*page header",
+        "type": "HTML",
+        "content": "<div class=\"navbar navbar-default navbar-static-top\">\r\n  <div class=\"container-fluid\">\r\n    <div class=\"navbar-header\">\r\n      <button type=\"button\" class=\"navbar-toggle collapsed\" data-toggle=\"collapse\" data-target=\"#navbar-collapse-1\">\r\n        <span class=\"sr-only\">Toggle navigation</span>\r\n          <span class=\"icon-bar\"></span>\r\n          <span class=\"icon-bar\"></span>\r\n          <span class=\"icon-bar\"></span>\r\n        </button>\r\n    </div>\r\n    <div class=\"collapse navbar-collapse\" id=\"navbar-collapse-1\">\r\n      <ul class=\"nav navbar-nav navbar-left\">\r\n        <li>[[/ | Home]]</li>\r\n        <li>[[/recent | Recent]]</li>\r\n      </ul>   \r\n      {{*navbox|navbar_left}}\r\n    \r\n      {{*account links|navbar_right}}\r\n    </div>\r\n  </div>\r\n</div>"
       }
     ]
   }
-}
```

Comment:
hard to see the diffs clearly here.  are there lots of CSS changes within the style cards or mostly just in how those cards or organized?